bindings/python/src/torrent_info.cpp

```diff
@@ -126,17 +126,23 @@ namespace
        return result;
     }
 
-    int get_tier(announce_entry const& ae) { return ae.tier; }
-    void set_tier(announce_entry& ae, int v) { ae.tier = v; }
-    int get_fail_limit(announce_entry const& ae) { return ae.fail_limit; }
-    void set_fail_limit(announce_entry& ae, int l) { ae.fail_limit = l; }
+    // Create getters for announce_entry data members with non-trivial types which need converting.
+    libtorrent::time_point get_next_announce(announce_entry const& ae) { return ae.next_announce; }
+    libtorrent::time_point get_min_announce(announce_entry const& ae) { return ae.min_announce; }
+    // announce_entry data member bit-fields.
     int get_fails(announce_entry const& ae) { return ae.fails; }
     int get_source(announce_entry const& ae) { return ae.source; }
     bool get_verified(announce_entry const& ae) { return ae.verified; }
     bool get_updating(announce_entry const& ae) { return ae.updating; }
     bool get_start_sent(announce_entry const& ae) { return ae.start_sent; }
     bool get_complete_sent(announce_entry const& ae) { return ae.complete_sent; }
     bool get_send_stats(announce_entry const& ae) { return ae.send_stats; }
+    void set_send_stats(announce_entry& ae, bool b) { ae.send_stats = b; }
```

Comment:
I don't believe these were ever meant to be accessible from the client and I believe I have removed them in master